solo/methods/base.py

```diff
@@ -356,9 +348,13 @@ def num_training_steps(self) -> int:
                 if dataset not in ["cifar10", "cifar100", "stl10"]:
                     data_dir = self.extra_args.get("data_dir", ".")
                     train_dir = self.extra_args.get("train_dir", "train")
-                    folder = os.path.join(data_dir, train_dir)
+                    folder = os.path.join(data_dir, str(train_dir))
+                    h5py_file = self.extra_args.get("train_h5_path", None)
```

Comment:
h5 only when not cifar or stl?

Reply:
I mean, you can convert both datasets to h5, but at the same time I have an if to filter just imagenet-100 classes in the h5 dataset, so I'm not sure.

Reply:
ah ok, mmmh what do you suggest?

Reply:
Also because for those datasets we already have the precomputed values